qt/scientific_interfaces/MultiDatasetFit/MultiDatasetFit.cpp

```diff
@@ -25,6 +28,49 @@ namespace {
 const int zoomToolPage = 0;
 const int rangeToolPage = 1;
 Mantid::Kernel::Logger g_log("MultiDatasetFit");
+
+/// Copy parameter values into a table workspace that is ready for
+/// plotting them against a dataset index.
+void formatParametersForPlotting(const Mantid::API::IFunction &function,
+                                 const std::string &parametersPropertyName) {
+  const auto &mdFunction =
+      dynamic_cast<const Mantid::API::MultiDomainFunction &>(function);
+
+  if (mdFunction.nFunctions() == 0) {
+    // Fit button was hit by mistake? Nothing to do here.
+    // A warning will be shown elsewhere.
+    return;
+  }
+
+  auto table =
+      Mantid::API::WorkspaceFactory::Instance().createTable("TableWorkspace");
+  auto col = table->addColumn("double", "Dataset");
+  col->setPlotType(1); // X-values inplots
+
+  auto nDomains = mdFunction.getNumberDomains();
+  assert(nDomains == mdFunction.nFunctions());
+
+  // Add columns for parameters and their errors
+  const auto &fun = *mdFunction.getFunction(0);
+  for (size_t iPar = 0; iPar < fun.nParams(); ++iPar) {
+    table->addColumn("double", fun.parameterName(iPar));
+    table->addColumn("double", fun.parameterName(iPar) + "_Err");
```

Comment:
[`FindEPP`](http://docs.mantidproject.org/nightly/algorithms/FindEPP-v2.html) produces very similar output table, and there the error column names are appended by `Error` instead of `_Err`. Would it make sense to be consistent with `FindEPP` here?

Reply:
`_Err` is consistent with the output from `PlotPeakByLogValue` algorithm which is used by the same interface to run sequential fits.